schema/CMakeLists.txt

```diff
@@ -1,97 +1,57 @@
-#
-# Copyright Soramitsu Co., Ltd. 2017 All Rights Reserved.
-# http://soramitsu.co.jp
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#        http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-compile_proto_to_cpp(block.proto)
-compile_proto_to_cpp(proposal.proto)
-compile_proto_to_cpp(primitive.proto)
-compile_proto_to_cpp(commands.proto)
-compile_proto_to_cpp(queries.proto)
-compile_proto_to_cpp(responses.proto)
-compile_proto_to_cpp(endpoint.proto)
-
-if (IROHA_ROOT_PROJECT)
-  compile_proto_to_grpc_cpp(endpoint.proto)
-  compile_proto_to_grpc_cpp(yac.proto)
-  compile_proto_to_grpc_cpp(ordering.proto)
-  compile_proto_to_grpc_cpp(loader.proto)
-  compile_proto_to_grpc_cpp(mst.proto)
-endif ()
-
-add_library(schema
-    block.pb.cc
-    commands.pb.cc
-    primitive.pb.cc
-    queries.pb.cc
-    responses.pb.cc
-    endpoint.pb.cc
-    proposal.pb.cc
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set(SM_SCHEMA_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../shared_model/schema/)
+set(SCHEMA_PATH ${SM_SCHEMA_PATH})
+compile_proto_only_grpc_to_cpp(endpoint.proto "-I${SM_SCHEMA_PATH}")
```

Comment:
Logically it should be SCHEMA_PATH?

Reply:
Not really, the model proto should be included for the building grpc services